analysis/analysis/ic50.py

```diff
@@ -0,0 +1,4 @@
+import numpy as np
```

Comment:
Perhaps it should go to `backend/` because it's part of the backend.

Reply:
I am not sure that it is good solution. For MVP maybe yes, but in the future as I understand we will have much more python code\
So, I just thought that calling it not from root folder is not a problem

Reply:
> For MVP maybe yes, but in the future as I understand we will have much more python code

And what's the problem? I mean `backend/analysis`. So we add just one folder to `backend` which I think is not a big deal, is it?

Reply:
So what do you think? No matter how much python code we will have, it will still be backend code, no?

Reply:
1. I don't want to see python code in stack project. For example in the future it will be big enough and it will be strange as for me
2. There are no advantages putting it in backend folder and also no disadvantages 
3. If edna will be big enough I think python part will be fully separate from haskell part. So, maybe I am wrong but I think it will work as a separate server. So, haskell will just call some api to `analysis` server and get result. In. this case, combining this things will be a little bit :ghc:

Reply:
> I don't want to see python code in stack project. For example in the future it will be big enough and it will be strange as for me

But then we can have `backend/server` and `backend/analysis` for example. Or `backend/haskell` and `backend/python`.

Anyway, I don't want to spend more time on this discussion, let's leave it as is and come back to it later when we have more python code.